test/suites/ACID.m

```diff
@@ -2434,6 +2435,26 @@
     set(ax4, 'Color', 'none');
 end
 % =========================================================================
+function [stat] = histogramPlot()
+
+  if ~exist('histogram')
```

Comment:
@egeerardyn , we introduced `isHG2()` in `m2t` quite recently. It checks for the new graphical system which implies a) MATLAB and b) at least R2014b. Since we ship it wit m2t it is existent and sufficient for the intented check.